framework/console/controllers/FixtureController.php

```diff
@@ -415,13 +415,36 @@ private function findFixtures(array $fixtures = [])
         $foundFixtures = [];
 
         foreach ($files as $fixture) {
-            $foundFixtures[] = basename($fixture, 'Fixture.php');
+            $relativeName = $this->getFixtureRelativeName($fixture);
+            $foundFixtures[] = $relativeName;
```

Comment:
no need to have a variable here, just do `$foundFixtures[] = $this->getFixtureRelativeName($fixture);`